src/queues/job-queue.js

```diff
@@ -151,7 +151,7 @@ const initQueue = () => {
   BetterSocialCronQueue.setCronCallback(generalDailyQueue, (job, done) =>
     BetterSocialCronQueue.process(job, done, {
       credderScoreQueue,
-      deleteActivityProcessQueue
+      removeActivityQueue
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The queue variable name in the `BetterSocialCronQueue` class has been changed from `deleteActivityProcessQueue` to `removeActivityQueue`. Make sure that `removeActivityQueue` is properly initialized and configured before being used here, otherwise it may cause unexpected behavior or errors.
```diff
- deleteActivityProcessQueue
+ removeActivityQueue
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->